server/views/pages/assessRecall.njk

```diff
@@ -24,26 +24,18 @@
             <dl class="govuk-summary-list">
                 <div class="govuk-summary-list__row">
                     <dt class=" govuk-summary-list__key govuk-!-width-one-half">
-                        First name
-                    </dt>
-                    <dd class=" govuk-summary-list__value govuk-!-width-one-half" data-qa='firstName'>
-                        {{ person.firstName }}
-                    </dd>
-                </div>
-                <div class="govuk-summary-list__row">
-                    <dt class=" govuk-summary-list__key govuk-!-width-one-half">
-                        Middle names
+                        Name
                     </dt>
-                    <dd class=" govuk-summary-list__value govuk-!-width-one-half" data-qa='middleNames'>
-                        {{ person.middleNames }}
+                    <dd class=" govuk-summary-list__value govuk-!-width-one-half" data-qa='name'>
+                        {{ person.firstName }} {{ person.lastName }}
```

Comment:
Is it not worth keeping middleNames in too?

Reply:
Just matching. it up to the prototype -
 https://hmpps-manage-recalls-prototype.herokuapp.com/assess/recall